### ISSUE TITLE
Evaluate other map layers

### ISSUE THREAD

**Issue by fsteeg** [hbz/lobid-organisations#174]
Test with Stamen Watercolor and German labels (see history of NWBib map).

Should be a unified approach for both maps, i.e. after #161 and #162 are done.

**Comment by fsteeg** [comment]
@SBRitter and I took a look at Stamen `watercolor`, `toner`, and `toner-lite` and agreed we should stick with the current tiles, which provide informative and familiar maps.

Added attribution for OSM to the current maps, deployed to staging. See:

http://test.lobid.org/organisations/search?q=k%C3%B6ln&format=html
http://test.lobid.org/organisations/DE-605?format=html

@SBRitter: As part of that, I removed the font-size setting in our CSS, as that caused the attribution text to be too large. I think the map popups look good with the smaller text. The issue you reported in https://github.com/hbz/lobid-organisations/issues/212 also doesn't seem to be back: http://test.lobid.org/organisations/DE-123?format=html
